src/components/todo/TodoList.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import { connect } from 'react-redux';
+import TodoListItem from "./TodoListItem";
+import NewTodoForm from "./NewTodoForm";
+import { removeTodo, markTodoAsCompleted } from "./actions";
+
+import  './TodoList.css';
+
+const TodoList = ({todos = [], onRemovePress, onCompletedPress}) => {
+    return (
+        <div className="list-wrapper">
+            <NewTodoForm />
+            {todos.map((todo, index) => <TodoListItem 
+                todo={todo} 
+                key={index}
```

Comment:
generally you want to try and avoid using the index as a key. Especially if the items can get reordered